### PR TITLE
[14.0][FW] server_environment_ir_config_parameter: Port PR from 12.0/13.0 to 14.0

### PR DESCRIPTION
**Pull request by sonhd91**
This PR migrate [PR#67](https://github.com/OCA/server-env/pull/67) from 12.0 to 14.0

### CODE REVIEW

File: server_environment_ir_config_parameter/__manifest__.py [hunk 1]
```diff
@@ -10,4 +10,9 @@
     "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/server-env",
     "depends": ["server_environment"],
+    'data': [
+        'views/view_ir_config_parameter.xml',
+    ],
+    'demo': [
```
Comment (simahawk) [inline server_environment_ir_config_parameter/__manifest__.py]: demo is not required

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
pre-commit is not happy

**Review by simahawk — CHANGES_REQUESTED** [review]
small things to fix

**Comment by sonhd91** [comment]
> pre-commit is not happy

Thanks, branch updated

**Review by simahawk — APPROVED** [review]

**Review by yvaucher — APPROVED** [review]

**Pull request merged**